app/javascript/packages/document-capture/components/document-capture.tsx

```diff
@@ -30,13 +31,28 @@ interface DocumentCaptureProps {
   onStepChange?: () => void;
 }
 
+const appRoot = document.getElementById('document-capture-form')!;
+
+function getDocAuthSeparatePagesEnabled() {
+  if (appRoot == null) {
+    return false;
+  }
+  if (appRoot.dataset == null) {
+    return false;
+  }
+
+  const { docAuthSeparatePagesEnabled } = appRoot.dataset;
+  return docAuthSeparatePagesEnabled === 'true';
+}
```

Comment:
This should ideally be passed in by the pack initializer.

See related: https://github.com/18F/identity-idp/blob/main/app/javascript/packages/README.md

>As much as possible, their behaviors should be reusable and make no assumptions about particular application pages or configuration. Instead, they should be initialized by a [pack](https://github.com/18F/identity-idp/blob/main/app/javascript/packs) which provides relevant configuration and page element references.

There's precedent for this:

https://github.com/18F/identity-idp/blob/1fc928c75cf70e261feeb220230b617722e7310a/app/javascript/packs/document-capture.tsx#L21-L44

We could add this as an existing property of one of the contexts defined in that file, create a new one, or just pass as a prop, depending whether leaf components will need to access this information.